### ISSUE TITLE
OTP e-mail provider down?

### ISSUE THREAD

**Issue by naton** [dexie/Dexie.js#1705]
Sorry if this is the wrong place for Dexie Cloud related hosting issues but I’ve been unable to login or share invites since sometime last week. Is it only me or are there problems with email delivery in general right now?

**Comment by dfahlander** [comment]
There seems to be a problem with some 3rd part spam blockers. I am still able to authenticate my gmail addresss but not my dexie address.

Looking into the blocks logs in sendgrid, I can see your email has been blocked a few times since April 18 with message "521 149.72.126.143 blocked by Spamcop - see https://www.spamcop.net/bl.shtml". The only other block I see is for my dexie.org email with the log entry "550 5.7.1 Service unavailable; client [149.72.123.24] blocked using csi.cloudmark.com".

I use Loopia as SMTP forwarder for my dexie.org email, and it seems to be the one using the blocker service. The reputation of dexie.cloud email is 100% and there are no spam reports reported back to sendgrid, so I do not know where these 3rd part blockers could possible make their decision on.

I'll try to see what is possible to do about it. FOr the time being, could you try using another email account and see if it comes through?

**Comment by naton** [comment]
Thanks 🙏🏻 I’ll try  that!

**Comment by dfahlander** [comment]
I'm in contact with sendgrid who had launched a new IP pool recently and they are trying to fix this.

**Comment by dfahlander** [comment]
Some providers are still blocking the shared SMTP email of dexie cloud. I send a new message to sendgrid today to excalate the issue. If this continues to be an issue, we will upgrade to a dedicated IP on sendgrid and move out from shared IP address.

**Comment by naton** [comment]
I got an OTP email last evening at 21:40 or so. Don't know which of my tests that caused this though 😅 so I couldn't use it. But felt promising 😃

**Comment by naton** [comment]
Mail are starting to go through now 🤞 At least when inviting via localhost.

**Comment by dfahlander** [comment]
Nice. I've approached sendgrid a few times about this. Maybe they've been in contact the blacklist provider to get the shared IP out from their list. As I said earlier - if this becomes a common problem, I'll step up the sendgrid subscription to dedicated IP.

**Comment by naton** [comment]
Unfortunately, I can't get email to work on [birdlist.netlify.app](https://birdlist.netlify.app/), only on localhost. I don't want you to step up in cost(?) just for me so if there's anything I can do, just let me know. 

I tried to get invites to show up in a list instead, for already logged in users, but that didn't work either (probably a different thing altogether though.. and surely something on my part) – just FYI if there's something related going on there.

**Comment by dfahlander** [comment]
There's been some issue with invite synkings that have been solved in the staging version and run for a week now. I just now published these changes to the production version of the server now, so it might be due to those issues.

The new server might fix the invite synking issue for you. There are still a few issues I need to debug this week but the current upgrade might be fixing the most important issues with invite synking. Emailing problem is separate though, but unfortunately, queruing invites in db.cloud.invites has had this synking bug also. I hope the publish I just did might solve that at least.

Also, try with the next version of the addon `dexie-cloud-addon@next` that has an issue about deleting data from left realms.
